zerver/openapi/zulip.yaml

```diff
@@ -16538,7 +16538,6 @@ paths:
             enum:
               - direct
               - stream
-              - private
```

Comment:
We need to add a note above `**Changes**: In Zulip 7.0 (feature level 174)` that `private` stopped being supported at this feature level. (I think you can use the same feature level for this commit as the previous one). And this should be mentioned in the API changelog too.

Reply:
<details><summary>set-typing-status</summary>
<img src="https://github.com/zulip/zulip/assets/56781761/d8fac7ca-17f4-461c-a46d-7e30ee819e28">
</img>
</details>

Reply:
Looks great!